enterprise/frontend/src/embedding-sdk/README.md

```diff
@@ -1224,6 +1224,19 @@ async function fetchRequestToken(url) {
 const config = defineEmbeddingSdkConfig({ fetchRequestToken });
 ```
 
+### Customizing loader and error components
```

Comment:
Note that we should have a PR soon that removes most of the content from README.md, and point to https://www.metabase.com/docs/latest/embedding/sdk/quickstart instead as the authoritative source of SDK documentation.